Sources/LibMobileCoin/ProtoExtensions.swift

```diff
@@ -186,3 +186,16 @@ extension FogLedger_BlockData {
         BlockMetadata(index: index, timestampStatus: timestampStatus)
     }
 }
+
+extension FogLedger_TxOutResult {
+    var timestampDate: Date? {
```

Comment:
Whats the purpose of this extension ?

Reply:
This is the same extension that exists is for [FogLedger_OutputResult](https://github.com/mobilecoinofficial/MobileCoin-Swift/blob/7c5dd2f5502c907ec82f92f414c4f38eb39b84a1/Sources/LibMobileCoin/ProtoExtensions.swift#L91) because the timestamp value is `UInt64.max` when it is not valid.